lib/rules/indent.js

```diff
@@ -608,9 +608,13 @@ module.exports = {
                 return elem !== null;
             });
 
-            // Skip if first element is in same line with this node
-            if (elements.length > 0 && elements[0].loc.start.line === node.loc.start.line) {
-                return;
+            // Skip if element is in same line with this node
+            if (elements.length > 0 && node.type === "ArrayExpression") {
+                elements.forEach(el => {
+                    if (el.loc.start.line === node.loc.start.line) {
+                        return;
+                    }
+                });
```

Comment:
Context: this is warned by dogfooding of `no-useless-return` rule #7441. Can I remove this `if` block?